add API3 under oracles.mdx (#99)
sei-protocol/sei-docs: pages/dev-ecosystem-providers/oracles.mdx

Oracles provide external data to smart contracts, enabling more dynamic and responsive applications. Notable oracles for Sei include:

## **Pyth**

- [Pyth Documentation](https://docs.pyth.network/home)

## **API3**

> The [API3 Market](https://market.api3.org/) provides access to 200+ price feeds on [Sei Mainnet](https://market.api3.org/sei) and [Testnet](https://market.api3.org/sei-testnet). The price feeds operate as a native push oracle and can be activated instantly via the Market UI.

Reviewer: ```suggestion
The [API3 Market](https://market.api3.org/) offers access to over 200 price feeds on both Sei [Mainnet](https://market.api3.org/sei) and [Testnet](https://market.api3.org/sei-testnet). These price feeds function as a native push oracle and can be activated instantly via the Market UI.
```

The price feeds are delivered by an aggregate of [first-party oracles](https://docs.api3.org/explore/airnode/why-first-party-oracles.html) using signed data and support [OEV recapture](https://docs.api3.org/explore/introduction/oracle-extractable-value.html).

> Unlike traditional data feeds, reading [API3 price feeds](https://docs.api3.org/guides/dapis/) enables dApps to auction off the right to update the price feeds to searcher bots which facilitates more efficient liquidation processes for users and LPs of DeFi money markets. The OEV recaptured is returned to the dApp.

Reviewer: ```suggestion
Unlike traditional data feeds, accessing [API3 price feeds](https://docs.api3.org/guides/dapis/) allows dApps to auction off the right to update the feeds to searcher bots, improving the efficiency of liquidation processes for users and LPs in DeFi money markets. The recaptured OEV is returned to the dApp.
```

Reviewer 2: I think `allows dApps to auction` should be `allow dApps to auction`
